shared/parcel.js

```diff
@@ -96,3 +97,14 @@ export function getParcelsNotIncluded(newParcels, allParcels) {
 export function hasTags(parcel) {
   return parcel && !utils.isEmptyObject(parcel.tags)
 }
+
+/**
+ * Check if parcel is listable or not
+ * @param parcel
+ * @return boolean - whether is listable or not
+ */
+export function isListable(parcel) {
+  return (
+    !isDistrict(parcel) && !isRoad(parcel) && !isPlaza(parcel) && parcel.owner
```

Comment:
I think only `isDistrict` is necessary here, as it only check for the existence of a `district_id` prop
```suggestion
    !isDistrict(parcel) && parcel.owner
```